app/gzac/src/main/resources/config/processlink/bezwaar.processlink.json

```diff
@@ -11,7 +11,11 @@
         "formDefinitionName": "user-task-validate-request",
         "processLinkType": "form",
         "formDisplayType": "panel",
-        "formSize": "medium"
+        "formSize": "medium",
+        "subtitles": [
```

Comment:
I wonder if this breaks the hash. But I think it was fixed by Ivo. Can you check with him?

Reply:
Normally the change to this specific file would break the hash, but with my change the DTO being changed shouldn't. But this is not a changelog import file so it doesn't matter either way.